src/main/java/use_cases/create_curriculum/CreateCurriculumInteractor.java

```diff
@@ -0,0 +1,22 @@
+package use_cases.create_curriculum;
+
+/**
+ * A use case interactor for the creation of a new curriculum
+ * Created: 11/27/2022
+ * Last updated: 11/29/2022
+ *
+ * @author MMachadoUofT
+ */
+public class CreateCurriculumInteractor implements CreateCurriculumInputBoundary {
+
+    /**
+     * Executes the use case for creating a new Curriculum
+     *
+     * @param name the new Curriculum's name
+     */
+    @Override
+    public void createCurriculum(String name) {
+
+    }
+
```

Comment:
Good thing going with your descriptions. It helps anyone to understand the broader view of everything.